### ISSUE TITLE
gnt-cluster verify complain about disk not in range, but cannot change setting

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#769]
Originally reported of Google Code with ID 713.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

> gnt-cluster (ganeti v2.9.3) 2.9.3
> Software version: 2.9.3
> Internode protocol: 2090000
> Configuration format: 2090000
> OS api version: 20
> Export interface: 0
> VCS version: v2.9.3

> hspace (ganeti) version v2.9.3
> compiled with ghc 7.4
> running on linux x86_64



<b>What distribution are you using?</b>
> Debian Wheezy+backports

<b>What steps will reproduce the problem?</b>
> "gnt-cluster verify" to check and get warning
> "gnt-cluster modify --specs-disk-size max=2t" for trying to change setting

<b>What is the expected output? What do you see instead?</b>
> Output from verify : WARNING: instance <instance-name>: disk-size/1 value 4194304 is not in range [1024, 1048576]
> Output from "gnt-cluster modify --specs-disk-size max=2t" : gnt-cluster: error: no such option: --specs-disk-size


<b>Please provide any additional information below.</b>

I got 2 instance with 2TB disks, gnt-cluster verify compains about it, according to a post on a mailing list I'm supposed to run "gnt-cluster modify --specs-disk-size max=2t" but that does not work.
```
Originally added on 2014-02-11 14:47:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-02-12 15:43:22 +0000 UTC.

Added Labels: Priority-Medium
Changed State: Accepted
Added to Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
The shared-file-storage dir in version 2.9. can only be changed at configure time (so before initializing the cluster). We changed that in 2.10 and offered the --shared-file-storage-dir option then. Unfortunately it was a bigger change why we did not backport it to 2.9. 

For --specs-disk-size, the option actually exist, but is named unintuitively and the documentation is not very clear either. We'll fix that.
```
Originally added on 2014-02-14 10:36:35 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Does that mean there is no way to use shared-file-storage with 2.9 ? that's annoying :/
```
Originally added on 2014-02-18 07:27:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
At least not without re-initializing the cluster, I'm afraid :/. If you did not have it enabled in a pre-2.9 cluster, you cannot enable it after an upgrade. But if you initialize a fresh cluster, you can use shared-file-storage (but not change the path after cluster init). Sorry!
```
Originally added on 2014-02-19 10:29:58 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Concerning the shared-storage directory, the question is already answered (and I'm sorry, you will have to upgrade to 2.10 if you want to change paths during the runtime of the cluster). Concerning the initial 'gnt-cluster modify' question, I believe the --ipolicy-bounds-specs option can be used.
```
Originally added on 2014-02-24 10:35:02 +0000 UTC.

Changed State: WontFix

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hello,

so I finally upgraded my cluster to 2.10.2 :


# gnt-cluster version
Software version: 2.10.2
Internode protocol: 2100000
Configuration format: 2100000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.10.2

but sadly I still cannot change shared storage dir

# gnt-cluster modify --shared-file-storage-dir=/vs_share/
Usage
=====
  gnt-cluster modify [opts...]

gnt-cluster: error: no such option: --shared-file-storage-dir

```
Originally added on 2014-04-22 08:44:17 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi!

I just double-checked and you are right, the option is not working properly. I sent a patch to fix it today. It will be released with the next 2.10 release. Sorry for your inconvenience and thanks for pointing it out!

I opened an extra bug for it, as this one is not originally about the option. See https://code.google.com/p/ganeti/issues/detail?id=811

Cheers,
Helga
```
Originally added on 2014-04-22 11:09:26 +0000 UTC.

Mentioned Issue with GoogleCode ID 811 has been migrated to GitHub with ID #867.
